src/app/(app)/(home)/search-filters/categories.tsx

```
@@ -0,0 +1,24 @@
+import { Category } from '@/payload-types';
+import { CategoryDropdown } from './category-dropdown';
+
+interface CategoriesProps {
+  data: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)